akka-bench-jmh/src/main/scala/akka/actor/ForkJoinActorBenchmark.scala

```diff
@@ -72,6 +72,81 @@ class ForkJoinActorBenchmark {
 
   @Benchmark
   @Measurement(timeUnit = TimeUnit.MILLISECONDS)
+  @OperationsPerInvocation(totalMessagesLessThanCores)
+  def pingPongLessActorsThanCores(): Unit = {
+    val pingPongs =
+      for {
+        i <- 1 to lessThanCoresActorPairs
+      } yield {
+        val ping = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
+        val pong = system.actorOf(Props[ForkJoinActorBenchmark.PingPong])
```

Comment:
I have fixed anyway, and some other cleanup in https://github.com/akka/akka/pull/22560/commits/4ce788220da4a00d3aa09941186af893de9170dc